mosec/protocol.py

```diff
@@ -92,6 +97,13 @@ def receive(self) -> Tuple[bytes, Sequence[bytes], Sequence[bytes]]:
                 len(ids),
                 struct.unpack("!" + "I" * len(ids), b"".join(ids)),
             )
+
+        if total_bytes > IPC_LARGE_DATA_SIZE:
+            warnings.warn(
+                f"IPC data ({total_bytes} bytes) is large,"
```

Comment:
Minor typo: need a space after the `,`? For further grammar correctness, can change `this` to `which` since one sentence cannot contain two subjects

Reply:
Fixed.